homeassistant/components/home_connect/api.py

```diff
@@ -56,6 +56,8 @@ def get_devices(self):
                 device = Dryer(self.hass, app)
             elif app.type == "Washer":
                 device = Washer(self.hass, app)
+            elif app.type == "WasherDryer":
+                device = WasherDryer(self.hass, app)
```

Comment:
This doesn't seem related to the light platform. Please don't mix different changes in the same PR. Make multiple PRs each focusing on one thing.

Reply:
Removed all none light platform related changes.